src/System.Runtime.Caching/src/Configurations.props

```diff
@@ -0,0 +1,8 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <BuildConfigurations>
+      netstandard;
```

Comment:
you should also have a netfx configuration (for a facade)

if S.R.Caching will compile against netstandard, and work on UWP and Linux, then you just need a netstandard configuration also. If it only works against Windows but not UWP, eg., then you need netstandard (as a backstop) and netcoreapp2.0

Reply:
This is Windows only for now.